internal/util/cmdutil/cmdutil.go

```diff
@@ -82,3 +84,18 @@ func ResolveAbsAndRelPaths(path string) (string, string, error) {
 
 	return relPath, absPath, nil
 }
+
+// DockerCmdAvailable runs `docker ps` to check that the docker command is
+// available, and returns an error with installation instructions if it is not
+func DockerCmdAvailable() error {
+	cmd := exec.Command("docker", "ps")
```

Comment:
Is there a particular reason you use `docker ps` which can potentially be expensive, instead of `docker version`?

Reply:
there is no reason, the choice was arbitrary. I've changed it to `docker version`, thank you for the suggestion